pages/map.vue

```diff
@@ -251,17 +273,56 @@ function initMap(geographicData) {
 function updateMap(geographicData) {
   // use d3 select and update to update the map
   // this lets us use transitions to fade in the new data
-  d3.select(mapSvg.value)
-    .selectAll('path')
-    .data(topojson.feature(geographicData, geographicData.objects['stanford-pp624tm0074-geojson']).features)
+  console.log("raw objects", geographicData.objects['stanford-pp624tm0074-geojson'])
+  // let geodata = topojson.feature(geographicData, geographicData.objects['stanford-pp624tm0074-geojson']).features
+  // Merge geometries so we end up with Oblast-level shapes.
+  const featureCollection = aggregate(
+    geographicData,
+    geographicData.objects['stanford-pp624tm0074-geojson'],
+    'name_1'
+  );
+
+  let map = d3.select(mapSvg.value)
+  let paths = map.selectAll('path')
```

Comment:
```suggestion
  const paths = map.selectAll('path')
```

When I see `let` I immediately start looking around to see where it's mutated. IMO `const` is simpler to comprehend if there are no reassignments.